mcp-server/src/templates/prompts/work_issue.yaml

```
@@ -159,6 +171,11 @@ template: |
   
   Create the pull request with the title and body defined above, ensuring it links to issue #{{issue}}.
   
+  After successfully creating the pull request, switch back to the default branch:
+  - Detect the default branch name (usually 'master' or 'main')
+  - Run: `git checkout [default-branch]`
+  - Confirm you're back on the default branch
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)